### PR TITLE
[4.5] docs(multiselect): Documentation for TagMode

### PR DESCRIPTION
**Pull request by ntacheva**

### CODE REVIEW

File: components/multiselect/tag-mode.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+---
+title: Tag Mode
+page_title: MultiSelect Tag Mode
```
Comment (yordan-mitev) [inline components/multiselect/tag-mode.md:3]:
```suggestion
page_title: MultiSelect - Tag Mode
```

File: components/multiselect/tag-mode.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+description: Explore the available tag modes in the Blazor MultiSelect.
```
Comment (yordan-mitev) [inline components/multiselect/tag-mode.md:4]: Starting with an action verb is great, just try adding more content specific to this article. Aim at around 150 symbols.

File: components/multiselect/tag-mode.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+The MultiSelect supports three possible configurations for displaying the tags:
+
+* [Single Mode](#single-mode)
+* [Multiple Mode (the default)](#multiple-mode)
+* [Summarized tags based on the number of selections](#summarized-tags-based-on-the-number-of-selections)
+
+## Single Mode
+
+When the single tag mode is enabled only one summary tag will be displayed showing the count of all the selected items.
```
Comment (yordan-mitev) [inline components/multiselect/tag-mode.md:25]:
```suggestion
When the single tag mode is enabled, only one summary tag will be displayed showing the count of all selected items.
```

File: components/multiselect/tag-mode.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+        Countries.Add("Serbia");
+        Countries.Add("Slovenia");
+        base.OnInitialized();
+    }
+}
+````
+
+## Multiple Mode
+
+When the multiple tag mode is enabled each selected item will be displayed as a separate tag. This is the default `TagMode` value (`MultiSelectTagMode.Multiple`).
```
Comment (yordan-mitev) [inline components/multiselect/tag-mode.md:64]:
```suggestion
When the multiple tag mode is enabled, each selected item will be displayed as a separate tag. This is the default `TagMode` value (`MultiSelectTagMode.Multiple`).
```

File: components/multiselect/tag-mode.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+        Countries.Add("North Macedonia");
+        Countries.Add("Montenegro");
+        Countries.Add("Serbia");
+        Countries.Add("Slovenia");
+        base.OnInitialized();
+    }
+}
+````
+
+## Summarized tags based on the number of selections
```
Comment (yordan-mitev) [inline components/multiselect/tag-mode.md:101]:
```suggestion
## Summarized Tags Based on the Number of Selections
```

**Review by yanisslav — APPROVED** [review]

File: components/multiselect/tag-mode.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+
+The Tag Mode of the MultiSelect controls how the selected items (tags) will be visualized in the main (input) element of the component. 
+
+To configure the tag mode, use the `TagMode` parameter. It accepts a member of the `Telerik.Blazor.MultiSelectTagMode` enum.
+
+The MultiSelect supports three possible configurations for displaying the tags:
+
+* [Single Mode](#single-mode)
+* [Multiple Mode (the default)](#multiple-mode)
+* [Summarized tags based on the number of selections](#summarized-tags-based-on-the-number-of-selections)
```
Comment (ntacheva) [inline components/multiselect/tag-mode.md:21]:
```suggestion
* [Summarized Tags Based on the Number of Selections](#summarized-tags-based-on-the-number-of-selections)
```

File: components/multiselect/tag-mode.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
+---
+title: Tag Mode
+page_title: MultiSelect - Tag Mode
+description: Explore the available tag modes in the Blazor MultiSelect.
```
Comment (ntacheva) [inline components/multiselect/tag-mode.md:4]:
```suggestion
description: Explore the available tag modes in the Blazor MultiSelect. See the options that the TagMode parameter of the MultiSelect supports and test it in combination with the MaxAllowedTags property.
```

**Pull request merged**